.github/workflows/build.yaml

```diff
@@ -39,7 +40,7 @@ jobs:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
```

Comment:
This is the only thing that didn't sit well at first glance, but I guess there is no other way around it.

Reply:
@ffilippopoulos yeah, same, I did mean to actually call this out.

I've used the new personal tokens that can target specific repos & actions, so it can only operate on the 2 repos it needs to. 

Only change here could be to use a token from yourself rather than me, so you can more easily manage it?

Reply:
Actually I may do that, so the commits appear from the bot